### PR TITLE
[CI:DOCS] Minor podmanimage docs updates.

### PR DESCRIPTION
**Pull request by cevich**
Discovered by review of
https://github.com/containers/buildah/pull/3200

Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

**Comment by cevich** [comment]
@TomSweeneyRedHat okay this one should be good now.  I manually verified clicking on the links now works properly.

**Comment by rhatdan** [comment]
LGTM
/approve

File: contrib/podmanimage/README.md [hunk 1]
```diff
@@ -17,22 +17,21 @@ default to `/`.
...
+    These images are built when a new Podman version becomes available in
     Fedora.  These images are intended to be unchanging and stable, they will
-    never be updated by automation once they've been pushed.  For build
-    details, see the configuration used to build it,
-    [podmanimage/stable/Dockerfile](stable/Dockerfile).
+    never be updated by automation once they've been pushed.  For build details,
+    please [see the configuration file](stable/Dockerfile).
   * `quay.io/containers/podman:latest` and `quay.io/podman/stable:latest` -
-    Built daily using the same Containerfile as above.  The podman version
+    Built daily using the same Dockerfile as above.  The podman version
```
Comment (TomSweeneyRedHat) [inline contrib/podmanimage/README.md:25]: sorry @cevich, missed this nit yesterday.
```suggestion
    Built daily using the same Dockerfile as above.  The Podman version
```

**Comment by TomSweeneyRedHat** [comment]
One last, then GTG

File: contrib/podmanimage/README.md (hunk 1 above)
Comment (cevich) [reply contrib/podmanimage/README.md]: woopsie-doodle.  Thanks.  Got it.

**Pull request merged**